pkgs/development/lua-modules/generated-packages.nix

```
@@ -1432,17 +1432,17 @@ moonscript = buildLuarocksPackage {
 };
 nvim-client = buildLuarocksPackage {
   pname = "nvim-client";
-  version = "0.2.0-1";
+  version = "0.2.2-1";
 
   src = fetchurl {
-    url    = https://luarocks.org/nvim-client-0.2.0-1.src.rock;
-    sha256 = "1ah9mjvz28hrbwnyb5n60znz3m0m41rn7jpnxwfx773cys3skidx";
+    url    = https://luarocks.org/nvim-client-0.2.2-1.src.rock;
+    sha256 = "0bgx94ziiq0004zw9lz2zb349xaqs5pminqd8bwdrfdnfjnbp8x0";
   };
   disabled = (luaOlder "5.1");
-  propagatedBuildInputs = [ lua mpack luv coxpcall ];
+  propagatedBuildInputs = [ super.lua mpack luv coxpcall ];
```

```suggestion
  propagatedBuildInputs = [ lua mpack luv coxpcall ];
```

That wasn't needed :)